copier/main.py

```diff
@@ -662,21 +695,23 @@ def run_update(self) -> None:
             print(
                 f"Updating to template version {self.template.version}", file=sys.stderr
             )
+        self.apply_update()
+
+    def apply_update(self):
+        conflict_method = self.conflict_method("apply_update")
+        if conflict_method:
+            conflict_method(self)
+            return
+
+        # Default implementation.
         # Copy old template into a temporary destination
         with TemporaryDirectory(
             prefix=f"{__name__}.update_diff."
         ) as old_copy, TemporaryDirectory(
             prefix=f"{__name__}.recopy_diff."
         ) as new_copy:
-            old_worker = replace(
```

Comment:
There are no real changes to this function. I mechanically extracted some code into separate functions and moved the `old_worker.run_copy()` up a bit. This makes the new code shorter and more similar to the old code.